source/pages/Start/Start.tsx

```diff
@@ -37,6 +41,8 @@ export const Start = () => {
   const onSubmit = async ({ password }: { password: string }) => {
     await unlock(password);
 
+    if (!isBitcoinBased) browser.runtime.sendMessage({ type: 'CHAIN_CHANGED' });
```

Comment:
This has the potential bug for resetting chainID and networkVersion to the syscoin state even in some cases where the first State after a user has unlocked the wallet is Polygon for example. Because it always runs into this section of the code:
https://github.com/syscoin/pali-wallet/blob/61b23f07f103ab38a9c0a2607a415f27eb394704/source/scripts/ContentScript/index.ts#L91